www/_posts/2016-03-12-plugin-release.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+author:
+    name: Carlos Santana
+    url: https://twitter.com/csantanapr
+title:  "Plugins Release"
+categories: news
+tags: release plugins
+---
+
+The following plugins were updated today:
+
+* cordova-plugin-camera@2.1.1
+* cordova-plugin-statusbar@2.1.2
+* cordova-plugin-globalization@1.0.3
+* cordova-plugin-splashscreen@3.2.1
+
+
+----
+You can update any plugin by removing it, and then re-adding it.
+
+ E.g. To update your media plugin:
+
+    cordova plugin rm cordova-plugin-camera --save
+    cordova plugin add cordova-plugin-camera@2.1.1 --save
+
+Changes include:
+<!--more-->
+
+cordova-plugin-camera@2.1.1
+* [CB-10825](https://issues.apache.org/jira/browse/CB-10825) android: Always request READ permission for gallery source
+* added apache license header to appium files
+* [CB-10720](https://issues.apache.org/jira/browse/CB-10720) Fixed spelling, capitalization, and other small issues.
+* [CB-10414](https://issues.apache.org/jira/browse/CB-10414) Adding focus handler to resume video when user comes back on leaving the app while preview was running
+* Appium tests: adjust swipe distance on ** Android **
```

Comment:
The spaces between `**` and `Android` cause it not to format properly, this happens in multiple places throughout the post